lib/SampleService/SampleServiceImpl.py

```diff
@@ -512,6 +515,37 @@ def update_sample_acls(self, ctx, params):
         #END update_sample_acls
         pass
 
+    def update_samples_acls(self, ctx, params):
+        """
+        Update the ACLs of many samples.
+        :param params: instance of type "UpdateSamplesACLsParams"
+           (update_samples_acls parameters. These parameters are the same as
+           update_sample_acls, except: ids - a list of IDs of samples to
+           modify.) -> structure: parameter "ids" of list of type "sample_id"
+           (A Sample ID. Must be globally unique. Always assigned by the
+           Sample service.), parameter "admin" of list of type "user" (A
+           user's username.), parameter "write" of list of type "user" (A
+           user's username.), parameter "read" of list of type "user" (A
+           user's username.), parameter "remove" of list of type "user" (A
+           user's username.), parameter "public_read" of Long, parameter
+           "at_least" of type "boolean" (A boolean value, 0 for false, 1 for
+           true.), parameter "as_admin" of type "boolean" (A boolean value, 0
+           for false, 1 for true.)
+        """
+        # ctx is the context object
+        #BEGIN update_samples_acls
+        _update_samples_acls(
+            params,
+            self._samples,
+            self._user_lookup,
+            ctx[_CTX_USER],
+            ctx[_CTX_TOKEN],
+            _AdminPermission.FULL,
+            ctx.log_info,
+        )
+        #END update_samples_acls
+        pass
```

Comment:
could delete all these unnecessary `pass` statements if you want to make linters happy *shrug*